aws/requests.yaml

```diff
@@ -1,4 +1,8 @@
 releases:
+    - name: "> 15.1.0"
```

Comment:
please make it > 15.2.0

we already have ready to merge release 15.2.0 that cannot have any extra changes

Reply:
ok